### ISSUE TITLE
`clean-checks-list` causes bad wrap

### ISSUE THREAD

**Issue by thockin** [refined-github/refined-github#8019]
### Description

The "identify feature" process failed to identify which feature would disable this difference.  Also notice how it linewraps "job failed".  I want to turn off these SHAs.

![checks_diff](https://github.com/user-attachments/assets/626762bc-de01-4f2e-ba8e-83252f0d7f09)

### How to replicate the issue + URL

https://github.com/kubernetes/kubernetes/pull/128374

### Extension version

24.10.28

### Browser(s) used

Chrome

### Token ⚠️

- [X] I set a token in the options page. I understand that some features require a token to work.

**Comment by busches** [comment]
It's from a CSS feature, which is not able to be disabled, so you can't identify it. You can override the css from it though.

Feature: https://github.com/refined-github/refined-github/blob/7a974686522a1fc1afd8ecf54cd8567e4cf63ce4/source/features/clean-checks-list.css

**Comment by fregante** [comment]
We can still look at why it's wrapping the line so early and whether we can avoid it. The current wrapping behavior is not intentional.

If you want to hide the SHA altogether you might be able to do so via CSS in Refined GitHub's CSS field

**Comment by fregante** [comment]
It seems that it's intentional: 

- https://github.com/refined-github/refined-github/issues/6945
- https://github.com/refined-github/refined-github/pull/7038


But this seems to be natively-implemented now, we can revert that PR:


<img width="648" alt="Screenshot" src="https://github.com/user-attachments/assets/75231be8-b399-408a-a346-1726f4cd98f7">

@Katsute do you want to take a look?

**Comment by thockin** [comment]
> You can override the css from it though.

For the benefit of the truly clueless (when it comes to CSS) - what exactly does that mean?  For me, those hashes are not interesting and the viertical space they waste is important.  How do I make them go away?

**Comment by thockin** [comment]
Thanks.  I understand a lot of things, but CSS is still magic to me.

**Comment by Katsute** [comment]
It's not wrapping properly because they are using a wide space ` ` [U+2001](https://www.compart.com/en/unicode/U+2001) instead of normal spaces. The CSS is treating the `succeeded.   
    BaseSHA` as a single word so it won't break there.

```html
<div class="color-fg-muted col-10 css-truncate css-truncate-target">
  <strong class="text-emphasized">
    pull-kubernetes-conformance-kind-ga-only-parallel
  </strong>

      

    —
      Job succeeded.                     BaseSHA:b7113809cb9d74b82749dc8e8e8c63f7e9a21890
</div>
```

Using `word-break: break-all` kinda fixes it, but only for this case.

![image](https://github.com/user-attachments/assets/2fe92a24-1c5d-4b7e-a9a6-0b0908088b6a)

Can't be used as a solution, would cause any long name to split in the middle of a word instead of moving the whole word to a new line.

![image](https://github.com/user-attachments/assets/a12cd394-5ec2-4356-8cee-a6cd57e1facc)

**Comment by fregante** [comment]
Thanks for digging into it! I think simply reverting https://github.com/refined-github/refined-github/pull/7038 should fix it, but it can only be done if GitHub already natively resolved https://github.com/refined-github/refined-github/issues/6945. From my screenshot it seems that they did, but I didn't check everywhere

**Comment by fregante** [comment]
Not a bug, as mentioned by Katsute, it appears that way due to the non-breaking spaces. Fix the CI job message to fix the wrapping. 

GitHub natively fixed #6945 in their new checks-list component and it has the same behavior

<img width="684" alt="Screenshot 12" src="https://github.com/user-attachments/assets/970906e2-0297-4156-81cd-e1ab227bedf0">
